package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "NodeJS-Mentoring-HW3",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon dist/app.js",
+    "build": "babel src --out-dir dist",
+    "bStart": "npx babel-node src/app.js"
```

Comment:
Any of the binaries loaded as dependencies and/or devDependencies are available in npm scripts directly (without any 3rd-parties like npx